gce-scripts/packages_to_install.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Installs all the necessary packages for pshtt to run.
+# Logs which package it is installing as well as it's success (0) or failure
+# (1).
+echo 'UPDATE'
```

Comment:
Rather than all the `echo ...` 's, would it be sufficient to make use of `set -x` bash command?